js/common.js

```diff
@@ -1389,7 +1389,7 @@ function flashIconButton(button, button_classes, icon_classes, duration) {
 function uniqid(prefix = "", more_entropy = false) {
     const sec = Date.now() * 1000 + Math.random() * 1000;
     const id = sec.toString(16).replace(/\./g, "").padEnd(14, "0");
-    return `${prefix}${id}${more_entropy ? `.${Math.trunc(Math.random() * 100000000)}`:""}`;
+    return `${prefix}${id}${more_entropy ? `.${Math.floor(10000000 + Math.random() * 90000000)}`:""}`;
```

Comment:
I agree, it would probably be more easier to understand.
```suggestion
    const suffix = more_entropy ? '.' + Math.floor(Math.random() * 100000000).toString().padStart(8, '0');
    return `${prefix}${id}${suffix}`;
```